README.md

```diff
@@ -68,10 +82,9 @@ The module is also available [on the OpenTofu registry](https://search.opentofu.
 
 ```hcl
 module "spacelift" {
-  source = "github.com/spacelift-io/terraform-aws-eks-spacelift-selfhosted?ref=v1.0.0"
+  source = "github.com/spacelift-io/terraform-aws-eks-spacelift-selfhosted?ref=<vX.Y.Z>"
```

Comment:
These are getting outdated all the time, so i thought I'll set this to a placeholder. There is one I wanted to keep as "normal" in the top of this file though.